### PR TITLE
Create new entities.

### PR DESCRIPTION
**Pull request by Finka95**
Create Entities
Remove WeatherForecast Controller
Remove WeatherForecast class

### CODE REVIEW

**Review by Genresal — APPROVED** [review]

**Review by CrazyKamenwik — APPROVED** [review]

File: TicketSystem/TicketSystem/Models/Message.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿namespace TicketSystem.Models
+{
+    public class Message
+    {
+        public int Id { get; set; }
+        public string? Text { get; set; }
+        public DateTime CreatedDate { get; set; }
```
Comment (5Ka-me) [inline TicketSystem/TicketSystem/Models/Message.cs:7]: Separate links and attributes

File: TicketSystem/TicketSystem/Models/Message.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        public User User { get; set; }
+        public Ticket Ticket { get; set; }
```
Comment (5Ka-me) [inline TicketSystem/TicketSystem/Models/Message.cs]: Nullable property

File: TicketSystem/TicketSystem/Models/Ticket.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿namespace TicketSystem.Models
+{
+    public class Ticket
+    {
+        public int Id { get; set; }
+        public ICollection<User> Users { get; set; }
+        public ICollection<Message>? Messages { get; set; }
+        public TicketStatus TicketStatus { get; set; }
```
Comment (5Ka-me) [inline TicketSystem/TicketSystem/Models/Ticket.cs]: Nullable property

File: TicketSystem/TicketSystem/Models/Ticket.cs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+﻿namespace TicketSystem.Models
+{
+    public class Ticket
+    {
+        public int Id { get; set; }
+        public ICollection<User> Users { get; set; }
```
Comment (5Ka-me) [inline TicketSystem/TicketSystem/Models/Ticket.cs]: Separate links and attributes

File: TicketSystem/TicketSystem/Models/User.cs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+﻿namespace TicketSystem.Models
+{
+    public class User
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
```
Comment (5Ka-me) [inline TicketSystem/TicketSystem/Models/User.cs]: Separate links and attributes

File: TicketSystem/TicketSystem/Models/User.cs [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+        public ICollection<Ticket>? Tickets { get; set; }
+        public UserRole Role { get; set; }
```
Comment (5Ka-me) [inline TicketSystem/TicketSystem/Models/User.cs]: Nullable property

**Review by 5Ka-me — CHANGES_REQUESTED** [review]

File: TicketSystem/TicketSystem/Models/Ticket.cs [hunk 7]
```diff
@@ -0,0 +1,11 @@
+﻿namespace TicketSystem.Models
+{
+    public class Ticket
+    {
+        public int Id { get; set; }
+        public TicketStatus TicketStatus { get; set; } = null!;
+
```
Comment (5Ka-me) [inline TicketSystem/TicketSystem/Models/Ticket.cs:7]: Move "TicketStatus" to other links

File: TicketSystem/TicketSystem/Models/User.cs [hunk 8]
```diff
@@ -0,0 +1,11 @@
+﻿namespace TicketSystem.Models
+{
+    public class User
+    {
+        public int Id { get; set; }
+        public string Name { get; set; } = null!;
+        public UserRole Role { get; set; } = null!;
+
+        public ICollection<Ticket>? Tickets { get; set; }
```
Comment (5Ka-me) [inline TicketSystem/TicketSystem/Models/User.cs:9]: Move "Role" to other links

**Review by 5Ka-me — APPROVED** [review]

**Pull request merged**